internal/cache/text.go

```diff
@@ -22,9 +22,9 @@ type fontSizeEntry struct {
 	style fyne.TextStyle
 }
 
-// GetFontSize looks up a calculated size required for the specified text parameters.
-func GetFontSize(text string, size float32, style fyne.TextStyle) (fyne.Size, float32) {
-	ent := fontSizeEntry{text, size, style}
+// GetFontMetrics looks up a calculated size and baseline required for the specified text parameters.
+func GetFontMetrics(text string, fontSize float32, style fyne.TextStyle) (size fyne.Size, base float32) {
+	ent := fontSizeEntry{text, fontSize, style}
 	fontSizeLock.RLock()
```

Comment:
`fontSizeEntry` should be renamed to `fontMetricEntry` too?

Reply:
I don't think so - the map is from font size metadata to font metric cache. The one you comment on here is on the left of that equation

Reply:
Oh I see, got it.